--- a/src/amd/vulkan/radv_wsi.c	2022-03-29 21:22:21.000000000 +0200
+++ a/src/amd/vulkan/radv_wsi.c	2022-04-09 11:48:16.387524228 +0200
@@ -93,7 +93,6 @@
    if (result != VK_SUCCESS)
       return result;
 
-   physical_device->wsi_device.supports_modifiers = physical_device->rad_info.chip_class >= GFX9;
    physical_device->wsi_device.set_memory_ownership = radv_wsi_set_memory_ownership;
    physical_device->wsi_device.get_prime_blit_queue = radv_wsi_get_prime_blit_queue;
    physical_device->wsi_device.signal_semaphore_with_memory = true;
